accepted/future-releases/constant-update-2018/feature-specification.md

```diff
@@ -0,0 +1,130 @@
+# Design Document for Constant Specification Update 2018.
+Author: lrn@google.com (@lrhn)
+
+## Motivation and Scope
```

Comment:
Can we give a concrete example of code that illustrates some of these issues?

Reply:
Done.